product/manager_test.go

```diff
@@ -170,8 +173,24 @@ func TestManagerPolling(t *testing.T) {
 	if pp1 == pp2 {
 		t.Errorf("number of products should have incremented")
 	}
+
+	ac := &auth.Context{
```

Comment:
The name `auth` is used in quota pkg instead of `ac`. Could we make it consistent?

Reply:
If it bothers you, sure. 😄

Reply:
It normally doesn't but I happened to see it :)

Reply:
It was everywhere, but it's all nice now.